javascript/ql/lib/change-notes/2023-11-27-cors-permissive-configuarion.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
```

Comment:
Oh yeah, this change-note should probably be in the `javascript/ql/src/change-notes` folder instead.